### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by phillipthelen**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[@phillipthelen](https://github.com/phillipthelen).
```
Comment (barrenechea) [inline CODE_OF_CONDUCT.md:63]: GitHub does not allow direct messaging to users. Maybe consider setting up a point of contact for this project? So you don't have to disclose PII

**Review by barrenechea — APPROVED** [review]
GTM! Great initiative 🚀

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (plygrnd) [reply CODE_OF_CONDUCT.md:63]: We could use confidential GitHub issues, but I'm not sure whether that'll work.

Comment (phillipthelen) [reply CODE_OF_CONDUCT.md:63]: I'll set up an email address for this on my server and will update the document then. Until then, my e-mail address is also in my GitHub profile

**Pull request merged**